Moved content to proper area. (#1361)
arXiv/arxiv-docs: source/help/endorsement.md

- you will need to update your email address to an institutional email.

### First time submissions 

> arXiv requires that users be endorsed before submitting their first paper to arXiv or a new category.

Reviewer: ```suggestion
```
Delete row

First time submitters to arXiv are encouraged to associate an institutional email address if they have one (see [author registration help](registerhelp.md)). This will expedite the endorsement process.

> Authors who are submitting papers to a subject category for the first time are required to get an endorsement from an established arXiv author. Detailed instructions on how to proceed with the endorsement request are provided at that time.

Reviewer: ```suggestion
Alternately you can seek personal endorsement from an established arXiv author. Detailed instructions on how to proceed with the endorsement request are provided at that time.
```
